test/cli/suggest_unsafe_dead/suggest_unsafe_dead.out

```diff
@@ -147,7 +147,6 @@ suggest_unsafe_dead.rb:81: This code is unreachable https://srb.help/7006
     suggest_unsafe_dead.rb:80: Replaced with `T.unsafe(self)`
     80 |    unless self
                    ^^^^
-  Got `A` originating from:
```

Comment:
The removals here seem unusual.  Were these expected?

Reply:
yeah, they're kindof useless because the list of locations explaining how we got `A` (which is already in the error message elsewhere) is empty. So it's redundant.